include/deal.II/base/config.h.in

```diff
@@ -138,6 +138,9 @@
  * cmake/modules/Find*.cmake
  */
 
+/* cmake/configure/configure_3_petsc.cmake */
```

Comment:
Can you change the comment to `/* cmake/modules/FindPETSC.cmake */ and put it in alphabetical order? :-)

Reply:
But this is set in [`configure_3_petsc.cmake`](https://github.com/dealii/dealii/blob/master/cmake/configure/configure_3_petsc.cmake#L139)?